.zuul.yaml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: dnf-run-tests
+    description: Run DNF tests
+    roles:
+      - zuul: zuul-distro-jobs
+    requires: repo
+#    require-projects:
+#      - rpm-software-management/ci-dnf-stack
+    branches:
```

Comment:
This is a branches matcher defintion. It means this job will get triggered only for branches: master.